policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/ApplicationConfiguration.java

```diff
@@ -29,80 +28,41 @@
 
 import uk.gov.gchq.palisade.jsonserialisation.JSONSerialiser;
 import uk.gov.gchq.palisade.service.policy.exception.ApplicationAsyncExceptionHandler;
-import uk.gov.gchq.palisade.service.policy.repository.BackingStore;
-import uk.gov.gchq.palisade.service.policy.repository.EtcdBackingStore;
-import uk.gov.gchq.palisade.service.policy.repository.HashMapBackingStore;
-import uk.gov.gchq.palisade.service.policy.repository.K8sBackingStore;
-import uk.gov.gchq.palisade.service.policy.repository.PropertiesBackingStore;
-import uk.gov.gchq.palisade.service.policy.repository.SimpleCacheService;
-import uk.gov.gchq.palisade.service.policy.service.CacheService;
-import uk.gov.gchq.palisade.service.policy.service.HierarchicalPolicyService;
+import uk.gov.gchq.palisade.service.policy.service.NullPolicyService;
+import uk.gov.gchq.palisade.service.policy.service.PolicyService;
+import uk.gov.gchq.palisade.service.policy.service.PolicyServiceCachingProxy;
+import uk.gov.gchq.palisade.service.policy.service.PolicyServiceHierarchyProxy;
 
-import java.net.URI;
-import java.util.Map;
-import java.util.Objects;
 import java.util.Optional;
 import java.util.concurrent.Executor;
 
-import static java.util.stream.Collectors.toList;
-
 /**
  * Bean configuration and dependency injection graph
  */
 @Configuration
-@EnableConfigurationProperties(CacheConfiguration.class)
 public class ApplicationConfiguration implements AsyncConfigurer {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationConfiguration.class);
 
-    @Bean
-    public CacheConfiguration cacheConfiguration() {
-        return new CacheConfiguration();
+    @Bean("nullService")
```

Comment:
should we also have the simplePolicyService as an impl bean?

Reply:
There is only a simple policy service in testing, should it be added to src?